integration-tests/bats/keyless.bats

```diff
@@ -827,7 +821,6 @@ SQL
 }
 
 @test "keyless: merge delete+add on two branches" {
-    skip_nbf_dolt_1 "conflicts resolve not implemented"
     dolt branch left
```

Comment:
Need a test in this file exercising the conflict cardinality logic you added

Reply:
I have an engine test covering it here:
https://github.com/dolthub/dolt/pull/3777/files#diff-51ab7d0a893924f00c2f31c46b1b61a64af3f1ea9a7a9ddda7cdf885d61094e1R1767